cmd/device-worker/main.go

```diff
@@ -74,7 +74,7 @@ func main() {
 	// Listen on the provided socket address.
 	l, err := net.Listen("unix", r.GetAddress())
 	if err != nil {
-		log.Fatal(err)
+		log.Fatalf("Cannot start listening on %s err: ", r.GetAddress(), err)
```

Comment:
I think there will be a failure in printing the message due to a mismatch between the number of expected argument in the format message (1) to the actual number or provided arguments (2)